bpf/lib/process.h

```diff
@@ -409,6 +456,77 @@ static inline __attribute__((always_inline)) void execve_map_delete(__u32 pid)
 	}
 }
 
+// execve_joined_info_map_error() will increment the map error counter
+static inline __attribute__((always_inline)) void execve_joined_info_map_error(void)
+{
+	int one = MAP_STATS_ERROR;
+	__s64 *cntr;
+
+	cntr = map_lookup_elem(&tg_execve_joined_info_map_stats, &one);
+	if (cntr)
+		*cntr = *cntr + 1;
+}
+
+static inline __attribute__((always_inline)) int
```

Comment:
The return value of this function is not used. I'd suggest either use it, or modify the function to return void.

Reply:
ok will fix it, thanks